packages/fermi/README.md

```diff
@@ -9,22 +9,22 @@
 
 <div align="center">
   <!-- Crates version -->
-  <a href="https://crates.io/crates/dioxus">
+  <a href="https://crates.io/crates/fermi">
     <img src="https://img.shields.io/crates/v/dioxus.svg?style=flat-square"
```

Comment:
Could you update the image links as well so that the version displayed matches the version of the package?

Reply:
Sure thing